### DISCUSSION TITLE
multichannel?

### DISCUSSION THREAD

**Discussion by ghost** [bluefireteam/audioplayers#1310] [Q&A]
I need to play multichannel files (up to 64, maybe more). It is possible?

**Comment by Gustl22** [comment]
What exactly do you mean by multichannel? Is there some sort of specification like "Dolby Surround"? You could run 64 audioplayers simultaneously...

**Reply by ghost** [reply]
A multichannel audio stream have 2 or more channels. Specs (like "Dolby Surrround") does not matter, and multiple stereo players would not work  for my use cases.

**Reply by Gustl22** [reply]
What is your file format then?

**Comment by ghost** [comment]
I notice that the backend for Linux is GStreamer; this is ideal because it supports many formats, but Gstreamer is only used for Linux (with this plugin). To answer your question, as examples: 4, or 6, or 8 channels (that would work for quadraphony, 5.1 and 7.1 speaker layouts), and I would need even more channels (16, 24, 32, 64)... For supported containers and codecs, the more the better (wav, aiff, mp3, aac, ogg, etc). If this plugin is designed for stereo, than I should look for something else.

**Reply by Gustl22** [reply]
I think this is a very rare and special use case. I don't think it will be supported. But you're welcome to fork the repo and try your best with the linux implementation.
